lms/djangoapps/course_api/blocks/tests/test_serializers.py

```diff
@@ -79,6 +83,7 @@ def assert_extended_block(self, serialized_block):
                 'id', 'type', 'lms_web_url', 'student_view_url',
                 'display_name', 'graded',
                 'block_counts', 'student_view_multi_device',
+                'lti_url'
```

Comment:
I might just be missing it, but I don't see any test that the returned lti url is actually what you expect.